app-rating/src/main/res/layout/star_button_layout.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This should probably be replaced with `<merge />` tag to avoid inflating two FrameLayouts. `android:orientation` is also redundant in FrameLayout?